docs/how-to/use-trained-models.txt

```diff
@@ -0,0 +1,188 @@
+.. _use-trained-models:
+
+Using Trained Checkpoints
+=================================
+
+Determined provides APIs for downloading trained checkpoints and loading them
+into memory in a python process.
+
+Querying Checkpoints
+--------------------
+
+The :class:`ExperimentReference <determined.experimental.ExperimentReference>`
+class contains the ``top_checkpoint`` method. Without arguments the method will
+check the experiment configuration searcher field for the ``metric`` and
+``smaller_is_better`` values. These values are used to sort the experiments
+checkpoints by validation performance. For example the following searcher
+configuration will result in `ExperimentReference.top_checkpoint` returning the
+checkpoint with the lowest "loss" metric.
+
+.. code:: yaml
+
+   searcher:
+     metric: "loss"
+     smaller_is_better: true
+
+The following snippet will return an instance of :class:`Checkpoint
+<determined.experimental.Checkpoint>` representing the checkpoint from an
+experiment with the best validation metric.
+
+.. code:: python
+
+  from determined.experimental import Experiment
+
+  checkpoint = Experiment(id).top_checkpoint()
+
+To sort checkpoints by another metric, pass ``smaller_is_better`` and
+``sort_by`` parameters to ``top_checkpoint`` where ``sort_by`` is the name of
+the metric to sort by and ``smaller_is_better`` defines whether to minimize or
+maximize the metric.
```

Comment:
Well, I can sort by any metric besides smaller_is_better right?

maybe clarify that by default it uses smaller_is_better ect.

Something like:
"Checkpoints can be sorted by any metric, by using "sort_by", which states which metric to use, and "smaller_is_better", which defines whether to minimize or maximize the metric."